content/en/docs/configuration/acme/dns01/google.md

```diff
@@ -122,3 +130,104 @@ spec:
+static service account credentials. The [workload identity
+how-to](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity)
+provides more detail on how workload identity functions, but briefly workload
+identity allows you to link a Google service accounts (GSA) to Kubernetes
+service accounts (KSA). This GSA/KSA linking is two-way, i.e., you must
+establish the link in GCP _and_ Kubernetes. Once configured, workload identity
+allows Kubernetes pods running under a KSA to access the GCP APIs with the
+permissions of the linked GSA. The workload identity how-to also provides
+detailed instructions on how to enable workload identity in your GKE cluster.
+The instructions in the following sections assume you are deploying cert-manager
+to a GKE cluster with workload identity already enabled.
+
+### Link KSA to GSA in GCP
+
+The cert-manager component that needs to modify DNS records is the pod created
+as part of the cert-manager deployment. The [standard methods for deploying
+cert-manger to Kubernetes](../../../installation/kubernetes.md) create the
+cert-manager deployment in the cert-manager namespace and its pod spec specifies
+it runs under the cert-mananger service account. To link the GSA you created
+above to the cert-manager KSA in the cert-manager namespace in your GKE cluster,
+run the following command.
+
+```bash
+$ gcloud iam service-accounts add-iam-policy-binding \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:$PROJECT_ID.svc.id.goog[cert-manager/cert-manager]" \
+    dns01-solver@$PROJECT_ID.iam.gserviceaccount.com
+```
+
+If your cert-manager pods are running under a different service account, replace
+`goog[cert-manager/cert-manager]` with `goog[NAMESPACE/SERVICE_ACCOUNT]`, where
+`NAMESPACE` is the namespace of the service account and `SERVICE_ACCOUNT` is the
+name of the service account.
+
+### Link KSA to GSA in Kubernetes
+
+After deploying cert-manager, add the proper workload identity annotation to the
+cert-manager service account.
+
+```bash
+$ kubectl annotate serviceaccount --namespace=cert-manager cert-manager \
+    "iam.gke.io/gcp-service-account=dns01-solver@$PROJECT_ID.iam.gserviceaccount.com"
+```
+
+Again, if your cert-manager pods are running under a different service account,
+replace `--namespace=cert-manager cert-manager` with `--namespace=NAMESPACE
+SERVICE_ACCOUNT`, where `NAMESPACE` is the namespace of the service account and
+`SERVICE_ACCOUNT` is the name of the service account.
+
+### Create an Issuer That Uses CloudDNS
+
+Next, create an `Issuer` (or `ClusterIssuer`) with a `clouddns` provider. An
+example `Issuer` manifest can be seen below with annotations. Note that the
+issuer does not include a `serviceAccountSecretRef` property. Excluding this
+instructs cert-manager to use the default credentials provided by GKE workload
+identity.
+
+```yaml
+apiVersion: cert-manager.io/v1alpha2
```

Comment:
```suggestion
apiVersion: cert-manager.io/v1
```

We're releasing v1 tomorrow with the v1 api so we're already keeping the docs up to date before merging